app/src/main/java/it/quip/android/actvitiy/LoginActivity.java

```diff
@@ -49,7 +49,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
 
         //noinspection SimplifiableIfStatement
```

Comment:
I'm going to go ahead and delete the login activity menu anyways. Not necessary and you can do what you need if you want to stick it back in there